openstudiocore/src/model/ElectricLoadCenterDistribution.cpp

```diff
@@ -479,8 +481,14 @@ namespace detail {
 
 
   // Storage Converter Object Name
-  //bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter);
-  //void ElectricLoadCenterDistribution_Impl::resetStorageConverter();
+  bool ElectricLoadCenterDistribution_Impl::setStorageConverter(const ElectricLoadCenterStorageConverter& converter) {
```

Comment:
Well the real right solution is to embed the storage converter (or any other true child object) into the ELCD.  This isn't really possible in IDF but could be done in JSON.

Switching the bussType automatically is great but it is complicated.  You have to decide which methods can change it and in which circumstances, then you'd have to document it all for users.  Totally fine to skip this for now.